includes/scripts.php

```diff
@@ -80,6 +80,7 @@ function wprss_register_scripts()
             'premiumSupport' => ( wprss_licensing_get_manager()->licenseWithStatusExists( License_Status::VALID ) )
         ));
 
+        wp_enqueue_script('lorem-embed-script', 'https://embed.asklorem.com/load.js', array(), null, true);
```

Comment:
I think we need to register then enqueue later

Reply:
done